docs/usage-examples/find.txt

```diff
@@ -0,0 +1,82 @@
+.. _laravel-find-usage:
+
+=======================
+Find Multiple Documents
+=======================
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: find many, retrieve, code example
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can retrieve multiple documents from a collection by calling the ``where()`` method
+on an Eloquent model or a query builder.
+
+Pass a query filter to the ``where()`` method to retrieve documents that meet a
+set of criteria. MongoDB returns the matching documents according to their
+:term:`natural order` in the database or according to the sort order that you can specify
+by using the ``orderBy()`` method.
+
+Example
+-------
+
+This usage example performs the following actions:
+
+- Uses the ``Movie`` Eloquent model to represent the ``movies`` collection in the
+  ``sample_mflix`` database
+- Retrieves and prints documents from the ``movies`` collection that match a query filter
+
+The example calls the following methods on the ``Movie`` model:
+
+- ``where()``: matches documents in which the value of the ``runtime`` field is greater than ``900``
+- ``orderBy()``: sorts matched documents by their ascending ``_id`` values
+- ``cursor()``: retrieves the query results by loading each ``Movie`` model into memory one at a time
```

Comment:
I think most of the time people want to use `->get()` to get a Laravel Collection. It's not a problem if there are few results, and it offers more features than a cursor (count, first, last ...).

Reply:
Sorry, I had suggested using the [cursor](https://github.com/mongodb/laravel-mongodb/pull/2833/files#r1556598455) since that's how we usually demonstrate this. Will update to use get().